### PR TITLE
jax.numpy: Added around function to jax frontend.

### PR DESCRIPTION
**Pull request by raeesm1200**

### CODE REVIEW

**Comment by raeesm1200** [comment]
Close #12357

File: ivy/functional/frontends/jax/numpy/mathematical_functions.py [hunk 1]
```diff
@@ -503,3 +503,8 @@ def conj(x, /):
 def subtract(x1, x2):
     x1, x2 = promote_types_of_jax_inputs(x1, x2)
     return ivy.subtract(x1, x2)
+
+
+@to_ivy_arrays_and_back
+def around(a, decimals=0, out=None):
+    return ivy.around(a, decimals, out)
```
Comment (Daniel4078) [inline ivy/functional/frontends/jax/numpy/mathematical_functions.py:510]: I do not think that there is a "ivy.around" function for you to call directly like that. Thank you

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
Hello, I found a potential problem with how you implement the function. Check the comment below.

File: ivy/functional/frontends/jax/numpy/mathematical_functions.py (hunk 1 above)
Comment (raeesm1200) [reply ivy/functional/frontends/jax/numpy/mathematical_functions.py]: I have fixed it, please have a look.

**Review by Daniel4078 — APPROVED** [review]
looks good to me! thanks

**Pull request merged**